packages/analytics-js-integrations/__tests__/integrations/GA4/browser.test.js

```diff
@@ -28,7 +28,7 @@ describe('Google Analytics 4 events tests', () => {
   let ga4;
   beforeEach(() => {
     ga4 = new GA4(
-      { measurementId: 'G-123456', extendPageViewParams: true, capturePageView: 'rs' },
+      { measurementId: 'G-123456', extendPageViewParams: true, capturePageView: 'rs', piiPropertiesToIgnore: [{piiProperty: ''}] },
```

Comment:
Add more test cases covering scenario where `sendUserTraitsAsPartOfInIt` is not present. 
Also add test where `piiPropertiesToIgnore` is not present
Basically negative test cases

Reply:
Separated all events tests (identify, track,page, group) and added different configuration combinations